### DISCUSSION TITLE
docker-compose-non-dev.yml file using a `-dev` image from an official release

### DISCUSSION THREAD

**Discussion by sfirke** [apache/superset#22245] [General]
One of the most common sources of confusion in the Superset Slack is from users who can't get Alerts & Reports to work.  I find myself suggesting the docker configuration I'm running in production:

- `apache/superset:2.0.0` image for the application container
- `apache/superset:2.0.0-dev` image for all other containers (I specify this once up top, as `x-superset-image:`

That way the worker container has the headless browser and drivers needed to take snapshots for Alerts & Reports.

Is this configuration best-practice or is there another approach folks would recommend instead?  This was the way I saw to run a stable release of the software while still having the necessary configuration to use Alerts & Reports and avoid building my own images.

It's meeting my needs and I'm curious whether there's a reason I wouldn't recommend this to others who are trying to get Alerts & Reports working using docker-compose.

**Comment by pankajsoni22** [comment]
That's a hack solution, which works fine.
But do you think, any person who wants to use these feature has to struggle to download two different images to run one single application.

Can't we reach on a solution where stable release itself has all the features available. And everything work out of the box using docker-compose-non-dev.yml 

Just thinking out loud !
